README.md

```diff
@@ -31,3 +31,60 @@ The version number of the "bleeding edge" in `master` is always a `-SNAPSHOT` ve
 
 These plugins are published to R3's [Artifactory](https://software.r3.com/artifactory/corda). More recently, they are also published to Gradle's own plugins
 repository and can be imported into your projects using Gradle's `plugins` DSL.
+
+The plugins themselves fall into two categories: those intended for developing CorDapps, and those which are primarily used to build Corda itself.
+
+### Plugins for CorDapp development.
+
+- [`net.corda.plugins.cordapp`](cordapp/README.md):
+This plugin packages your CorDapp classes into a single jar file, along with
+the contants of all of the jar's dependent jars. Dependencies which are added
+to Gradle's `cordaCompile`, `cordaRuntime` and `cordapp` configurations are
+excluded from this packaging. The final jar is then signed.
```

Comment:
What will happen if `cordapp` entry is omitted , will a JAR contain any default values for `name`/`vendor`/`targetPlatformVersion` etc?
In other words is this obligatory:
```
cordapp {
    info {
        name "Trader Demo"
        vendor "R3"
        targetPlatformVersion corda_platform_version.toInteger()
        minimumPlatformVersion 1
    }
}
```

Reply:
Also maybe point out: 
The detailed documention of ``cordapp`` and ``cordformation`` plugins is available on https://docs.corda.net/cordapp-build-systems.html. (?)

Reply:
This is already mentioned by [the `cordapp` plugin's own README](cordapp/README.md).